packages/design-tokens/src/tokens/functional/components/timeline/base.json

```diff
@@ -0,0 +1,26 @@
+{
+  "brand": {
+    "Timeline": {
+      "lineHeight": {
+        "value": "var(--brand-text-lineHeight-300)"
+      },
+      "lineLength": {
```

Comment:
for consistency, we use `maxWidth` on `Card`, and the value is `50ch`

Reply:
This is the maximum line length of the text within the component, not the maximum width of the component itself.